### PR TITLE
docs: section summary pages

### PR DESCRIPTION
**Pull request by juliusmarminge**
Assists #879

https://create-t3-app-git-julius-pages-t3-oss.vercel.app/en/usage
https://create-t3-app-git-julius-pages-t3-oss.vercel.app/en/deployment

cc @asrvd

### CODE REVIEW

**Comment by asrvd** [comment]
can we do something similar for the usage route?

**Comment by juliusmarminge** [comment]
> can we do something similar for the usage route?

yes needs more styling too just a POC

**Comment by juliusmarminge** [comment]
> can we do something similar for the usage route?

Done https://create-t3-75zhuzzfr-t3-oss.vercel.app/en/usage

File: www/src/pages/en/deployment/index.astro [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+import Layout from "../../../layouts/docs.astro";
+import IndexPage from "../../../components/docs/indexPage.astro";
+import { Frontmatter, SIDEBAR } from "../../../config";
+import { getLanguageFromURL } from "../../../languages";
+
+const frontmatter: Frontmatter = {
+  title: "Deployment",
```
Comment (c-ehrlich) [inline www/src/pages/en/deployment/index.astro:8]: how will this work with translation? could we stick the frontmatter in markdown files for each language with a fallback to english, or something like that?

Comment (juliusmarminge) [reply www/src/pages/en/deployment/index.astro:8]: This file will be in all langs so when they copy the `en` folder they'll just have to translate this too. Not a whole lot

File: www/src/components/docs/indexPage.astro [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (juliusmarminge) [inline www/src/components/docs/indexPage.astro:1]: propose a better name for this component xd

**Comment by asrvd** [comment]
it's getting an unneeded padding-left as it's an `ul` element, maybe we should change them to just divs or section -> divs
image

**Comment by asrvd** [comment]
can i commit on this branch directly🤔

**Comment by juliusmarminge** [comment]
> can i commit on this branch directly🤔

removed the padding.

yes feel free to commit here

**Comment by asrvd** [comment]
> > can i commit on this branch directly🤔
> 
> removed the padding.
> 
> yes feel free to commit here

do I have write perms here though?

**Comment by juliusmarminge** [comment]
> > > can i commit on this branch directly🤔
> > 
> > 
> > removed the padding.
> > yes feel free to commit here
> 
> do I have write perms here though?

added you

**Comment by juliusmarminge** [comment]
Should we add the pagination component here too or is that overkill?

CleanShot 2022-11-27 at 16 56 26@2x

**Comment by asrvd** [comment]
> Should we add the pagination component here too or is that overkill?
> 
> CleanShot 2022-11-27 at 16 56 26@2x

can add

**Comment by juliusmarminge** [comment]
> > Should we add the pagination component here too or is that overkill?
> 
> > 
> 
> > CleanShot 2022-11-27 at 16 56 26@2x
> 
> 
> 
> can add

Just checked and we'd have to do some rearchitecting on the sidebar config for that to work so let's skip that for now.

**Comment by asrvd** [comment]
I think we can merge this one then 🤔

**Comment by juliusmarminge** [comment]
@c-ehrlich @nexxeln review pls 🙏🏼

**Comment by juliusmarminge** [comment]
Fk it I'll just admin merge this. These pages aren't really public yet anyways, only way to reach them is by entering the URL

**Pull request merged**